pages/common/exiftool.md

```diff
@@ -5,11 +5,11 @@
 
 - Print the EXIF metadata for a given file:
 
-`exiftool {{file}}`
+`exiftool {{path/to/file}}`
 
 - Remove all EXIF metadata from the given files:
 
-`exiftool -All= {{file1 file2 ...}}`
+`exiftool -All={{path/to/file1 path/to/file2 ...}}`
```

Comment:
Isn't the ` ` required?

```suggestion
`exiftool -All= {{path/to/file1 path/to/file2 ...}}`
```

Reply:
I don't know, but when `=` is used I expect the value to be joined with an option name.

Reply:
```

       exiftool -all= dst.jpg
            Delete all meta information from an image.  Note: You should NOT do
            this to RAW images (except DNG) since proprietary RAW image formats
            often contain information in the makernotes that is necessary for
            converting the image.

```
[There](https://manned.org/exiftool.1) is a space after this option. Accepting your suggestion.